code/game/verbs/suicide.dm

```diff
@@ -58,7 +59,7 @@
 							"[src] is holding \his breath! It looks like \he's trying to commit suicide.")
 
 		visible_message("<span class='danger'>[suicide_message]</span>", "<span class='userdanger'>[suicide_message]</span>")
-
+		suiciding = 1

```

Comment:
You can't put `suiciding = 1` this low, otherwise it will only be set if the person suicides without holding anything. Either put it back where it was and set it back to `0` on the failure block above, or just set it to 1 up there as well